### PR TITLE
refactor(question): addrouter and component top

### PR DESCRIPTION
**Pull request by jojo051**
add router and component top

### CODE REVIEW

File: src/components/Questions.jsx [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+import { connect } from 'react-redux';
+import {
+  Link,
+} from 'react-router-dom';
+import BorderTopCard from './BorderTopCard';
 
-function Questions({ props }) {
-  const [question, setQuestion] = useState(null);
+function Questions(props) {
+  const [questions, setQuestion] = useState(null);
```
Comment (rocambille) [inline src/components/Questions.jsx]: start with an empty array instead of null ;)

File: src/components/Questions.jsx [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
     <>
+      <div>
+        <Link to="/boxes/1">
+          <BorderTopCard />
+        </Link>
+      </div>
       <img className="imageQuestion" src="" alt="" />
       <p>{}</p>
+      <div>
+        {questions !== null ? questions.map(question => <p>{question.content}</p>) : ''}
```
Comment (rocambille) [inline src/components/Questions.jsx]: with an empty array, you can remove the conditional: just do the map

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/components/Questions.jsx [hunk 3]
```diff
@@ -1,25 +1,44 @@
...
+import { connect } from 'react-redux';
+import {
+  Link,
+} from 'react-router-dom';
+import BorderTopCard from './BorderTopCard';
 
-function Questions({ props }) {
-  const [question, setQuestion] = useState(null);
+function Questions(props) {
+  const [questions, setQuestion] = useState();
```
Comment (rocambille) [inline src/components/Questions.jsx]: you are here starting with undefined. start with an empty array:

    const [questions, setQuestion] = useState([]);

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/components/Questions.jsx (hunk 3 above)
Comment (jojo051) [reply src/components/Questions.jsx]: j'ai miss click

**Pull request merged**